### PR TITLE
Review manual and update

### PR DESCRIPTION
**Pull request by bas-ie**
This began as a move of all info subdirectories back into the main tree, at Jaime's suggestion here:

https://trello.com/c/neb1tL4G/11-move-the-subdirectories-of-important-course-information-into-the-same-directory-as-everything-else-so-that-all-the-links-e-g-our

However, there were many areas that hadn't been kept up to date. Even our Phase 0 rollback email addresses included one from Dev Bootcamp!

It's a slightly annoying PR to review because most directories have been renamed. Let me know if I can clarify anything.

Points covered:
1. Spell check, naming consistency, formatting.
2. Occasional grammar fixes.
3. Updating any information that implies rolling cohorts to suggest static cohorts.
4. Updating email addresses.
5. Fix broken links.
6. Re-label 3 agreements as 3 pillars, since that seems to be our current name for them.
7. Remove numbers from directory names, because it makes them too brittle (all links break when we change the order of something).

### CODE REVIEW

File: pairing/README.md [hunk 1]
```diff
@@ -22,20 +22,18 @@ That said pairing varies between teams.  Some pair 99% of the time; some pair on
 Pairing [doesn't feel natural or easy for everyone](http://blogs.atlassian.com/2009/06/pair_programming_is_kryptonite/). We ask that you keep an open-mind on pairing throughout EDA and challenge yourself to work on your pairing skills while here.
```
Comment (locksmithdon) [inline pairing/README.md]: _open mind_ instead of _open-mind_?

File: three-pillars/README.md [hunk 2]
```diff
@@ -20,21 +20,21 @@ At Dev Academy, we have codes of conduct that support an optimal learning enviro
 
 Part of being in integrity also means that when you fall out of it for whatever reason (such as being a human being) you own up to it.  Most of the time all it takes is an acknowledgment of being out of integrity and everybody can move on.
 
-## The second agreement: Kindness
+## Kindness
 We also expect every person to be kind to one another. Kindness does not mean "nice;" they are actually not the same thing. Being kind does mean being friendly, generous, and considerate, but it does not preclude giving constructive, truthful feedback as long as it is <strong>A</strong>ctionable, <strong>S</strong>pecific, and <strong>K</strong>ind (ASK).
```
Comment (locksmithdon) [inline three-pillars/README.md]: I don't think we need to bold the first letter of each word, but even if we do, we can use markdown.

File: use-of-space/README.md [hunk 3]
```diff
@@ -1,12 +1,15 @@
 # Use of the space
 
-While Phase-0 is a remote phase, Phase-0 students are welcome to use the space during office hours (9am - 5.30pm). We have a limited number of desks free, and you will need to bring your own laptop. 
+While Phase 0 is a remote phase, Phase 0 students are welcome to use the space during office hours (9am - 5.30pm). We have a limited number of desks free, and you will need to bring your own laptop. 
+
 
 ## What support is available @ Dev Academy?
```
Comment (locksmithdon) [inline use-of-space/README.md]: @ :arrow_right: _at_?

**Review by don-smith — CHANGES_REQUESTED** [review]
Looks like this was long overdue. Nice work (as always)!

**Comment by richchurcher** [comment]
Thanks! Always a few I miss ;)

**Review by don-smith — APPROVED** [review]

**Pull request merged**